examples/llama/generator.py

```diff
@@ -137,30 +149,63 @@ def run(self, prompt: str):
                 break
             output_ids.append(next_token)
 
+        elapsed = self.runtime.stop()
+        print(f"elapsed {elapsed:.5f} ms, itr {len(output_ids)}")
         output_text = self.tokenizer.decode(output_ids)
         return output_text
 
 
 def worker(args: argparse.Namespace, rank: int):
+    os.environ["RANK"] = str(rank)
+    os.environ["LOCAL_RANK"] = str(rank)
+
     def log(msg):
         print(f"[Rank {rank}] {msg}")
 
     with open(args.params_path, "r") as f:
         params = json.load(f)
 
+    prompt_list = ["Where is the capital of France?"]
+    if args.only_torch_model:
+        generator = Llama.build(
+            ckpt_dir=args.ckpt_dir,
+            tokenizer_path=args.tok_path,
+            max_seq_len=1024,
+            max_batch_size=1,
+        )
+        generation_tokens = generator.generate(
+            prompt_tokens=[
+                generator.tokenizer.encode(
+                    f"[INST] {prompt} [/INST]", bos=True, eos=False
+                )
+                for prompt in prompt_list
+            ],
+            max_gen_len=1024,
+            temperature=0,
+            top_p=0.9,
+            logprobs=False,
+            echo=False,
+        )
+        output_text = [
+            {"generation": generator.tokenizer.decode(t)}
+            for t in generation_tokens
+        ]
+        if rank == 0:
+            log(f"{output_text}")
+        return
+
     gen = Generator(
-        ModelArgs7B(**params),
+        ModelArgs13B(**params),
```

Comment:
I think just using `ModelArgs` is better to avoid confusion. Arguments will actually just follow `params`, not `ModelArgs7B` or `ModelArgs13B`